packages/augur-state/src/Controller.ts

```diff
@@ -0,0 +1,20 @@
+import { DB } from './db/DB';
```

Comment:
Should this Controller be used from both the server and the Worker in the browser as our main entrypoint?

If so we need one more layer here that is a Server.ts or whatever that starts the controller and then starts the server listeners.